scipy/linalg/flapack_other.pyf.src

```diff
@@ -546,6 +546,28 @@ subroutine <prefix>trtrs(lower, trans, unitdiag, n, nrhs, a, lda, b, ldb, info)
 end subroutine <prefix>trtrs
 
 
+subroutine <prefix>trcon(norm, uplo, diag, n, a, lda, rcond, work, irwork, info)
+
+    ! ?TRCON estimates the reciprocal of the condition number of a
+    ! triangular matrix A, in either the 1-norm or the infinity-norm.
+
+    callstatement (*f2py_func)(norm,uplo,diag,&n,a,&lda,&rcond,work,iwork,&info);
```

Comment:
This `iwork` needs to be `irwork`. That's why tests are failing.

Reply:
```suggestion
    callstatement (*f2py_func)(norm,uplo,diag,&n,a,&lda,&rcond,work,irwork,&info);
```

Reply:
OK, I fixed that so you can see the failures I was talking about.